docs/src/main/sphinx/sql/set-time-zone.rst

```diff
@@ -0,0 +1,61 @@
+=============
+SET TIME ZONE
+=============
+
+Synopsis
+--------
+
+.. code-block:: text
+
+    SET TIME ZONE LOCAL
+    SET TIME ZONE expression
+
+Description
+-----------
+
+Sets the current default time zone displacement for the current SQL-session.
+
+If the ```LOCAL`` option is specified, the time zone for the current SQL-session
+will be set to the original time zone of the current SQL-session.
+
+If the ``expression`` option is specified:
+
+- if the type of the ``expression`` is a string, the time zone for the current SQL-session is set to the corresponding region-based time zone ID or the corresponding zone offset.
+- if the type of the ``expression`` is an interval, the time zone for the current SQL-session is set to the corresponding zone offset relative to UTC. It must be in the range of [-14,14] hours.
+
+
+Examples
+--------
+
+Use the default time zone for the current SQL-session::
+
+    SET TIME ZONE LOCAL;
+
+Use a region-based time zone identifier for specifying the time zone::
+
+    SET TIME ZONE 'America/Los_Angeles';
+
+Same outcome can be obtained when the time zone identifier is the output
+of a function call::
+
+    SET TIME ZONE concat_ws('/', 'America', 'Los_Angeles');
+
+Use a zone offset for specifying the time zone::
```

Comment:
This and the next example should be the first ones in the list of examples since they are more reliable imho.

Reply:
You're right.
I used the textual IDs as they are easier to remember for me